src/controller/CHIPDeviceController.cpp

```diff
@@ -2018,9 +2063,13 @@ void DeviceCommissioner::PerformCommissioningStep(DeviceProxy * proxy, Commissio
         // Read the requested minimum connection times from all network commissioning clusters
         readPaths[7] = app::AttributePathParams(app::Clusters::NetworkCommissioning::Id,
                                                 app::Clusters::NetworkCommissioning::Attributes::ConnectMaxTimeSeconds::Id);
+        // Read the current fabrics
+        readPaths[8] = app::AttributePathParams(app::Clusters::OperationalCredentials::Id,
+                                                app::Clusters::OperationalCredentials::Attributes::Fabrics::Id);
```

Comment:
Again, I think all the `app::Clusters` prefixes should be able to go away....

Reply:
fixed